examples/contrib/oed/ab_test.py

```diff
@@ -0,0 +1,112 @@
+import torch
+import numpy as np
+import matplotlib.pyplot as plt
+
+import pyro
+import pyro.distributions as dist
+from pyro import optim
+from pyro.contrib.oed.eig import vi_ape
+
+
+# Set up regression model dimensions
+N = 100  # number of participants
+p_treatments = 2  # number of treatment groups
+p = p_treatments  # number of features
+prior_stdevs = torch.Tensor([1, .5])
+
+softplus = torch.nn.Softplus()
+
+
+def model(design):
+    # Allow batching of designs
+    loc_shape = list(design.shape)
+    loc_shape[-2] = 1
+    loc = torch.zeros(loc_shape)
+    scale = prior_stdevs
+    # Place a normal prior on the regression coefficient
+    # w is 1 x p: hence use .independent(2)
+    w_prior = dist.Normal(loc, scale).independent(2)
+    w = pyro.sample('w', w_prior).transpose(-1, -2)
+
+    with pyro.iarange("map", N, dim=-2):
+        # Run the regressor forward conditioned on inputs
+        prediction_mean = torch.matmul(design, w).squeeze(-1)
+        # y is an n-vector: hence use .independent(1)
+        pyro.sample("y", dist.Normal(prediction_mean, 1).independent(1))
+
+
+def guide(design):
```

Comment:
Add a comment mentioning that this variational family includes the true posterior